lib/Zonemaster/LDNS.pm

```diff
@@ -40,7 +40,8 @@ Returns the ldns version string. Can be exported, but is not by default.
 
 =item to_idn($name, ...)
 
-Takes a number of strings and returns a list of them converted to IDNA format.
+Takes a number of strings and returns a list of them in the ASCII-Compatible Encoding
+form (A-label) of the strings.
```

Comment:
Maybe change "the ASCII-Compatible Encoding form (A-label)" to "A-label form (https://tools.ietf.org/html/rfc5890#section-2.3.2.1)" or something similar? To me that would be clearer.